kinescope/kinescope.py

```diff
@@ -0,0 +1,110 @@
+"""This XBlock embeds content from Kinescope through Iframes"""
+
+import pkg_resources
+from web_fragments.fragment import Fragment
+from xblock.core import XBlock
+from xblock.fields import Integer, Scope, String
+try:
+    from xblock.utils.studio_editable import StudioEditableXBlockMixin
+except ModuleNotFoundError: # For compatibility with Palm and earlier
+    from xblockutils.studio_editable import StudioEditableXBlockMixin
+from xblock.validation import ValidationMessage
+
+
+from .utils import _, is_url, parse_valid_kinescope_url
+
+
+class KinescopeXBlock(StudioEditableXBlockMixin, XBlock):
+    """
+    This XBlock renders Iframe for Kinescope videos.
+    """
+
+    display_name = String(
+        display_name=_("Display name"),
+        default=_("Kinescope"),
+        scope=Scope.settings,
+        help=_("Display name for this XBlock."),
+    )
+
+    video_link = String(
+        display_name="Video Link/URL",
+        default="",
+        scope=Scope.content,
+        help=_(
+            "Video link copied from Kinescope dashboard. The video id below is extracted "
+            "from this link if provided"
```

Comment:
```suggestion
            "from this link if provided."
```

Tiny nit - it just doesn't match the convention of other `help` arguments.